### ISSUE TITLE
Embedding mistral.rs in Rust - Build fail with features=["metal"]

### ISSUE THREAD

**Issue by andrewlimmer** [EricLBuehler/mistral.rs#769]
I am trying to embed mistral in a rust app with metal acceleration, but there is a build failure while building with the feature = ["metal"] enabled in my cargo.toml file.  

## Minimum reproducible example
Cargo.toml file
```
[dependencies]
mistralrs = { git = "https://github.com/EricLBuehler/mistral.rs.git", features = ["metal"]}
```

Build
```
 cargo test -- --nocapture 
```

## Error
What is the error? If the error is very long, please try to extract an excerpt of it and post a link to a [Gist](https://gist.github.com/)
There is an error within the candle-core library
<img width="883" alt="image" src="https://github.com/user-attachments/assets/f7ec0966-a98f-4d8b-bf85-0535db09861a">

## Other information
Please specify:
- MacOS
-  M1
- GPU or accelerator information
    - Metal Mac M1

## Latest commit or version
version=v0.3.0

**Comment by EricLBuehler** [comment]
@andrewlimmer I added #771 which should fix this, can you please confirm?

**Comment by andrewlimmer** [comment]
> @andrewlimmer I added #771 which should fix this, can you please confirm with:
> 
> ```
> git pull
> git switch seed_fix
> ```

This fixed the build issue.
Thank you so much!

**Comment by EricLBuehler** [comment]
@andrewlimmer thank you, I merged #771. Please feel free to reopen.
